### PR TITLE
[Sprint 8] add html meta tags

### PR DESCRIPTION
**Pull request by stevahnes**

### CODE REVIEW

File: src/assets/logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....262 351.795 C 171.611 352.145,176.979 354.000,177.598 353.986 C 177.929 353.979,176.670 353.440,174.800 352.789 C 171.281 351.564,170.933 351.466,171.262 351.795 M218.200 352.000 C 217.696 352.217,217.622 352.344,218.000 352.344 C 218.330 352.344,218.960 352.189,219.400 352.000 C 219.904 351.783,219.978 351.656,219.600 351.656 C 219.270 351.656,218.640 351.811,218.200 352.000 M214.200 353.200 C 212.880 353.604,212.042 353.950,212.338 353.967 C 213.011 354.008,217.200 352.805,217.200 352.572 C 217.200 352.317,216.834 352.393,214.200 353.200 M179.200 354.353 C 181.014 354.976,184.160 355.556,186.800 355.753 L 189.400 355.948 187.000 355.568 C 185.680 355.358,183.790 355.020,182.800 354.815 C 179.648 354.163,178.051 353.958,179.200 354.353 M208.800 354.449 C 207.810 354.641,205.471 354.992,203.603 355.230 C 201.734 355.467,200.114 355.756,200.003 355.870 C 199.496 356.392,207.142 355.334,210.600 354.404 C 212.178 353.980,211.079 354.007,208.800 354.449 M192.117 356.297 C 193.611 356.374,195.951 356.374,197.317 356.296 C 198.682 356.218,197.460 356.155,194.600 356.155 C 191.740 356.156,190.622 356.220,192.117 356.297 " stroke="none" fill="#6ca4d4" fill-rule="evenodd"></path></g></svg>
```
Comment (stevahnes) [inline src/assets/logo.svg:1]: We can change the image with the proper NgResume Logo once we create it! :)

File: src/app/app.component.ts [hunk 2]
```diff
@@ -8,20 +9,48 @@ import { takeUntil } from 'rxjs/operators';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent implements OnDestroy {
-  title = 'ng-resume';
+export class AppComponent implements OnInit, OnDestroy {
+  title = 'NgResume';
   activeId = '';
 
+  private readonly description =
```
Comment (zeyao) [inline src/app/app.component.ts]: We could move this to a constant class, can do later

**Review by zeyao — APPROVED** [review]
LGTM

File: src/app/app.component.ts (hunk 2 above)
Comment (stevahnes) [reply src/app/app.component.ts]: Modified haha.. :)

**Pull request merged**